Create ui-c8y-1021-54-4-tracking-view-in-device-management-seems-to-be-broken-multiple-issues.md (#3094)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1021-54-4-tracking-view-in-device-management-seems-to-be-broken-multiple-issues.md

---
date: ""

> title: Issues in Tracking tab in Device Management have been fixed

Reviewer: ```suggestion
title: Issues in the Tracking tab in the device details have been fixed
```

> title: Tracking view in Device Management seems to be broken multiple issues

Reviewer 2: ```suggestion
title: Issues in Tracking tab in Device Management have been fixed
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-4358
version: 1021.54.4
---

> Tracking view in Device Management seems to be broken multiple issues

Reviewer 2: ```suggestion
In **Tracking** tab realtime updates of device position were not activated even though the **Realtime** button seemed activated. This has been fixed and now tracking updates both device position and tracking path in realtime. Changing the time range and refreshing the list had no effect. This has been fixed and users can filter events by different time ranges and reload the list.
```

> In **Tracking** tab realtime updates of device position were not activated even though the **Realtime** button seemed activated. This has been fixed and now tracking updates both device position and tracking path in realtime. Changing the time range and refreshing the list had no effect. This has been fixed and users can filter events by different time ranges and reload the list.

Reviewer: ```suggestion
In the **Tracking** tab in the device details the device position was not updated in realtime even though the **Realtime** button was activated. This has been fixed and now the device position and the tracking path are both updated in realtime. Moreover, changing the time range and refreshing the list had no effect. This has been fixed and users can again filter events by different time ranges.
```
